api/instance/v1/instance_sdk.go

```diff
@@ -2063,21 +2063,21 @@ func (s *API) GetImage(req *GetImageRequest, opts ...scw.RequestOption) (*GetIma
 
 type CreateImageRequest struct {
 	Zone scw.Zone `json:"-"`
-
+	// Name name of the image
```

Comment:
double name

Reply:
I open an internal ticket to fix the documentation generation.